core/src/test/scala/org/http4s/rho/ApiTest.scala

```diff
@@ -194,31 +194,33 @@ class ApiTest extends CatsEffectSuite {
     val r2 = Gone("Foo")
     val c2 = H[`Content-Length`].captureMapR(_ => Left(r2))
     val v1 = ruleExecutor.runRequestRules(c2.rule, req)
-
-    v1 match {
-      case r: FailureResponse[IO] =>
-        (
-          r.toResponse.map(_.status),
-          r2.map(_.resp.status)
-        ).parTupled.map(x => assertEquals(x._1, x._2))
-
-      case _ =>
-        fail("Unexpected runRequestRules result found")
-    }
-
     val c3 = H[Accept].captureMapR(Some(r2))(_ => ???)
     val v2 = ruleExecutor.runRequestRules(c3.rule, req)
 
-    v2 match {
-      case r: FailureResponse[IO] =>
-        (
-          r.toResponse.map(_.status),
-          r2.map(_.resp.status)
-        ).parTupled.map(x => assertEquals(x._1, x._2))
+    for {
+      _ <- v1 match {
+        case r: FailureResponse[IO] =>
+          IO.raiseError(new Throwable("hui"))
```

Comment:
```suggestion
```
This looks like it was added in, but I'm not sure why?  It doesn't appear to serve a purpose here?

Reply:
Ouch. That was only for debugging purposes. I'm so sorry about that :(

Reply:
No problem.   Happens to all of us!